src/test/debuginfo/pretty-std.rs

```diff
@@ -116,17 +115,18 @@
 // NOTE: OsString doesn't have a .natvis entry yet.
 
 // cdb-command: dx some
-// cdb-check:some             : Some [Type: enum$<core::option::Option<i16>>]
+// cdb-check:some             : Some [Type: enum$<core::option::Option<i16> >]
 // cdb-check:    [...] variant$         : Some (0x1) [Type: core::option::Option]
 // cdb-check:    [...] __0              : 8 [Type: short]
 
 // cdb-command: dx none
-// cdb-check:none             : None [Type: enum$<core::option::Option<i64>>]
+// cdb-check:none             : None [Type: enum$<core::option::Option<i64> >]
 // cdb-check:    [...] variant$         : None (0x0) [Type: core::option::Option]
 
 // cdb-command: dx some_string
 // NOTE: cdb fails to interpret debug info of Option enums on i686.
-// cdb-check:some_string      [Type: enum$<core::option::Option<alloc::string::String>, 1, [...], Some>]
+// cdb-check:some_string      : Some({...}) [Type: enum$<core::option::Option<alloc::string::String>, 1, [...], Some>]
+// cdb-check:    [...] __0              : "IAMA optional string!" [Type: alloc::string::String]
```

Comment:
This case seems to fail on CI. The [output](https://github.com/rust-lang-ci/rust/runs/2962304803#step:25:11485) looks like the removed line. Does this look familiar, @wesleywiser? I remember you mentioning problems with natvis not working on CI.

Reply:
This looks to me like the issue I ran into where cdb doesn't visualize niche-enums correctly but WinDbg and VS do. I ended up just validating that the debug info was correct and not testing the visualizer (using the `!` format specifier).